narwhals/_dask/expr.py

```diff
@@ -445,6 +450,16 @@ def clip(
             returns_scalar=False,
         )
 
+    def head(self) -> NoReturn:
+        # We can't (yet?) allow methods which modify the index
+        msg = "`Expr.head` is not supported for the Dask backend. Please use `LazyFrame.head` instead."
```

Comment:
Quick heads up: sadly DaskDataframe also doesn't implement `head` (nor `tail`) 🥲

Reply:
🫠  Where can I check that? I only found Dask API reference for Arrays and DataFrames

Reply:
`LazyFrame` will point to `DaskLazyFrame` for the dask backend, and the message is suggesting to use such method, so I guess you can check if `narwhals/_dask/dataframe.py`?

Reply:
![image](https://blogger.googleusercontent.com/img/b/R29vZ2xl/AVvXsEglhazxkK13mGC6xgRlAxX7yu10gEXM3vvzEGFlt_8cozYSoBrWwoEo5K0iYBG5lnstHdOgiwl8tI2vOpWZ3wPFKL63Mzm0tLvjnN3tdwoQv99zMHwMd7QVYH6n_8DmPYRWjmfwlf44pjUC/s0/tumblr_8f6fcd7ab5170b194baacc0416cbb846_9eaf9a12_500.gif)